contracts/forger_stake_delegation/README.md

```diff
@@ -107,6 +122,21 @@ In order to run the withdraw operation you will need to set the following enviro
 - `OWNER_PRIVATE_KEY` - Optional. If not set, it will be the same as PRIVATE_KEY. If OWNER_SIGNED_MESSAGE is set, it will be ignored. 
 - `OWNER_SIGNED_MESSAGE` - Optional. If the owner has already signed the message, paste the signature here, it will be used instead of signing the message with OWNER_PRIVATE_KEY. If not, leave it empty and the script will sign the message.
 
+#### StakeOf script
```

Comment:
I would give a small description what this script actually does. Like "this script ......."